tests/Elasticsearch/Tests/ConnectionPool/SniffingConnectionPoolTest.php

```diff
@@ -424,10 +418,9 @@ public function testTen_TimeoutNine_SniffTenth_AddTwoDead_TimeoutEveryone()
         ];
         $connectionPool = new SniffingConnectionPool($connections, $selector, $connectionFactory, $connectionPoolParams);
 
-        $retConnection = $connectionPool->nextConnection();
-        $this->assertSame($newConnections[11], $retConnection);
```

Comment:
the diff is a bit confusing here, the first `$retConnection = $connectionPool->nextConnection();` throws an exception, so the rest was a dead code